app/components/DappRequest/MessageSuccess.jsx

```diff
@@ -9,30 +9,32 @@ import CheckMarkIcon from '../../assets/icons/confirm-circle.svg'
 
 type Props = {
   isVerify: boolean,
+  text?: string,
 }
 
-const MessageSuccess = ({ isVerify }: Props) => (
-  <FullHeightPanel
-    headerText="Wallet Connect"
-    renderCloseButton={() => <CloseButton routeTo={ROUTES.DASHBOARD} />}
-    renderHeaderIcon={() => (
-      <div>
-        <WallletConnect />
+const MessageSuccess = ({ isVerify, text }: Props) => {
```

Comment:
Remove the `isVerify` parameter and change who is using this component to pass the whole text as parameter